src/package/nuspec/TestPlatform.TestHost.nuspec

```diff
@@ -36,6 +44,107 @@
     <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\x86\msdia140.dll" target="lib\netstandard1.5\x86\" />
     <file src="Microsoft.TestPlatform.TestHost\netcoreapp1.0\x64\msdia140.dll" target="lib\netstandard1.5\x64\" />
 
+    <!-- UWP -->
+    <file src="Microsoft.TestPlatform.TestHost\netstandard1.4\testhost.dll" target="lib\uap10.0\" />
+    <file src="Microsoft.TestPlatform.TestHost\netstandard1.4\Microsoft.TestPlatform.CommunicationUtilities.dll" target="build\uap10.0\" />
```

Comment:
Can we add to lib and set as developmentDependency?

Reply:
moved to lib